packages/tasks/src/model-libraries.ts

```diff
@@ -446,6 +446,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.pyannote_audio,
 		filter: true,
 	},
+	"py-feat": {
+		prettyLabel: "py-feat",
```

Comment:
This is for display purposes. I've seen both `Py-FEAT` (in the [repo](https://github.com/cosanlab/py-feat)) and `Py-Feat` (in the [docs](https://py-feat.org/pages/intro.html)).

Reply:
Thanks, will add